### PR TITLE
[Doc] Update README

### PR DESCRIPTION
**Pull request by Anonymitaet**

### CODE REVIEW

**Comment by Anonymitaet** [comment]
@freeznet could you please help review? Thanks

File: README.md [hunk 1]
```diff
@@ -1,12 +1,14 @@
 # AWS Lambda connector
 
-The [AWS Lambda](https://aws.amazon.com/lambda/) sink connector is a [Pulsar IO connector](http://pulsar.apache.org/docs/en/next/io-overview/) for sending messages from Apache Pulsar to Amazon AWS Lambda.
+The [AWS Lambda](https://aws.amazon.com/lambda/) sink connector is a [Pulsar IO connector](http://pulsar.apache.org/docs/en/next/io-overview/) for pulling data from Pulsar topics and persists data to AWS Lambda to invoke Lambda functions.
```
Comment (freeznet) [inline README.md:3]:
```suggestion
The [AWS Lambda](https://aws.amazon.com/lambda/) sink connector is a [Pulsar IO connector](http://pulsar.apache.org/docs/en/next/io-overview/) for pulling messages from Pulsar topics to AWS Lambda to invoke Lambda functions. 
```

Comment (freeznet) [reply README.md:3]: AWS lambda is not provide as a data persistence service

File: docs/aws-lambda-sink.md [hunk 2]
```diff
@@ -21,9 +21,9 @@ owner_img: "/images/streamnative.png"
 id: "aws-lambda-sink"
 ---
 
-The [AWS Lambda](https://aws.amazon.com/lambda/) sink connector is a [Pulsar IO connector](http://pulsar.apache.org/docs/en/next/io-overview/) for sending messages from Apache Pulsar to Amazon AWS Lambda.
+The [AWS Lambda](https://aws.amazon.com/lambda/) sink connector is a [Pulsar IO connector](http://pulsar.apache.org/docs/en/next/io-overview/) for pulling data from Pulsar topics and persists data to AWS Lambda to invoke Lambda functions.
```
Comment (freeznet) [inline docs/aws-lambda-sink.md:24]:
```suggestion
The [AWS Lambda](https://aws.amazon.com/lambda/) sink connector is a [Pulsar IO connector](http://pulsar.apache.org/docs/en/next/io-overview/) for pulling messages from Pulsar topics to AWS Lambda to invoke Lambda functions. 
```

**Review by freeznet — APPROVED** [review]
LGTM, except one change about data persistence.

**Pull request merged**